index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "multi-download" {
+	/**
+	 * Download multiple files at once
+	 * @param urls The urls to download
+	 * @param options The options
+	 * @param options.rename The function to rename the file
+	 * @param options.delayTime The delay time between each download
+	 */
+	export default async function multiDownload(
+		urls: string[],
+		options?: {
+			rename?: (options: {
+				url: string;
+				index: number;
+				urls: string[];
+			}) => string;
+			delayTime?: number;
```

Comment:
The name does not make it clear that it's a delay between each download.

You also need to add it to the readme.